src/foam/box/sf/SF.js

```diff
@@ -342,9 +344,12 @@ foam.CLASS({
                                     .build();
 
             journalMap_.put(getSimpleFilename(filename), journal);
-            if ( journal.getFileOffset() == journal.getFileSize() ) continue;
+            if ( journal.getFileOffset() == journal.getFileSize() ) {
+              ((DAO) x.get("eventRecordDAO")).put(new EventRecord(getX(), this, "SAF file complete", "Id: " + getId() + ", file: " + getSimpleFilename(filename), LogLevel.INFO, null));
```

Comment:
```suggestion
              ((DAO) x.get("eventRecordDAO")).put(new EventRecord(getX(), this, "SAF file complete", getId(), null, file: " + getSimpleFilename(filename), LogLevel.INFO, null));
```
assuming getId is the host, it can be used for the partnerid,

Reply:
yes, id is host, will update